Source/WebCore/accessibility/AXTextMarker.h

```diff
@@ -99,10 +101,43 @@ struct TextMarkerData {
     {
         return ObjectIdentifier<AXIDType>(objectID);
     }
+
+    SafeTextMarkerData toSafeTextMarkerData() const;
+
 private:
     void initializeAXIDs(AXObjectCache&, Node*);
 };
 
+// Safer version of TextMarkerData with a WeakPtr for Node.
+// TextMarkerData uses a raw pointer for Node because it is
+// used with memset / memcmp.
```

Comment:
Looks to be required for converting to and from AXTextMarkerRef but I'm not an accessibility expert. Maybe there is a way to get rid of the unsafe version completely. I was planning to follow-up to rely on the safe version as much as possible in WebKit but was planning to keep using the unsafe version for converting to and from AXTextMarkerRef (because I do not know how to avoid it).